test/matlab_experimental/ctTestThermo.m

```diff
@@ -0,0 +1,482 @@
+            Zh = self.phase.elementalMassFraction('H');
+            Zar = self.phase.elementalMassFraction('Ar');
+
+            exp1 = 0.5 + 0.5 * (15.999 / 18.015);
+            exp2 = 0.5 * (2.016 / 18.015);
+            exp3 = 0.0;
+
+            self.assumeEqual(Zo, exp1, 'AbsTol', self.atol);
+            self.assumeEqual(Zh, exp2, 'AbsTol', self.atol);
+            self.assumeEqual(Zar, exp3, 'AbsTol', self.atol);
+
+            self.getInvalidValue('elementalMassFraction', {'C'}, 'No such element');
+            self.getInvalidValue('elementalMassFraction', {5}, 'No such element');
+        end
+
+        function testWeights(self)
+            aw = self.phase.atomicMasses;
+            mw = self.phase.molecularWeights;
+
+            self.assumeEqual(length(aw), self.phase.nElements);
+            self.assumeEqual(length(mw), self.phase.nSpecies);
+
+            for i = 1:length(mw)
+                testWeight = 0.0;
+                for j = 1:length(aw)
+                    testWeight = testWeight + ...
+                                 aw(j) * self.phase.nAtoms(i, j);
+                end
+                self.assumeEqual(testWeight, mw(i), 'RelTol', self.rtol);
+            end
+        end
+
+        function testCharges(self)
+            chargePhase = Solution('gri30_ion.yaml', 'gas');
+            charges = chargePhase.charges;
+            test = {{'E',-1}, {'N2',0}, {'H3O+',1}};
+
+            for i = 1:length(test)
+                species = test{i}{1};
+                charge = test{i}{2};
+
+                flag = sum(ismember(chargePhase.speciesNames, species));
+                self.assumeGreaterThan(flag, 0);
+                
+                idx = chargePhase.speciesIndex(species);
+                self.assumeEqual(charges(idx), charge);
+            end
+            clear chargePhase
+        end
+
+        function testReport(self)
+            % Fails because report method is missing. 
+            self.assumeFail();
+
+            str = self.phase.report;
+
+            self.assumeSubstring(str, self.phase.phaseName);
+            self.assumeSubstring(str, 'temperature');
+
+            for i = 1:self.phase.nSpecies
+                name = self.phase.speciesName(i);
+                self.assumeSubstring(str, name{:});
+            end
+        end
+
+        function testRefInfo(self)
+            self.assumeFail();
+
+            self.assumeEqual(self.phase.refPressure, OneAtm, 'RelTol', self.rtol);
+            self.assumeEqual(self.phase.minTemp, 300, 'RelTol', self.rtol);
+            self.assumeEqual(self.phase.maxTemp, 3500, 'RelTol', self.rtol);
+        end
+
+        function testSingleGetters(self)
+            self.assumeFail();
+
+            val = self.phase.T;
+            exp = 300;
+            self.assumeEqual(val, exp, 'RelTol', self.rtol);
+            self.assumeGreaterThan(self.phase.maxTemp, 0);
+            self.assumeGreaterThan(self.phase.minTemp, 0);
```

Comment:
These seem less useful than the tests of `maxTemp` and `minTemp` above in `testRefInfo`.